patterns.hpp

```diff
@@ -144,90 +144,8 @@ namespace patterns
 	#define CONCATENATE1(arg1, arg2) arg1 ## arg2
 	#define CONCATENATE(arg1, arg2) CONCATENATE1(arg1, arg2)
 
-	/*
-	 * Concatenate with empty because otherwise the MSVC preprocessor
-	 * puts all __VA_ARGS__ arguments into the first one.
-	 */
```

Comment:
Could you move this comment into the relevant place of the python script so the arcane knowledge isn't lost?